homeassistant/components/sensor/tibber.py

```diff
@@ -33,14 +35,18 @@
 @asyncio.coroutine
 def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
     """Set up the Tibber sensor."""
-    import Tibber
-    tibber = Tibber.Tibber(config[CONF_ACCESS_TOKEN],
-                           websession=async_get_clientsession(hass))
-    yield from tibber.update_info()
-    dev = []
-    for home in tibber.get_homes():
-        yield from home.update_info()
-        dev.append(TibberSensor(home))
+    import tibber
+    tibber_connection = tibber.Tibber(config[CONF_ACCESS_TOKEN],
+                                      websession=async_get_clientsession(hass))
+
+    try:
+        yield from tibber_connection.update_info()
+        dev = []
+        for home in tibber_connection.get_homes():
+            yield from home.update_info()
+            dev.append(TibberSensor(home))
+    except (asyncio.TimeoutError, aiohttp.ClientError):
+        raise PlatformNotReady
```

Comment:
raise PlatformNotReady()

Reply:
Why?
No other components seems to do that?